interview-questions.md

```diff
@@ -4,58 +4,91 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer: The method super() is used in order to pass inheritance between parent and child classes.
 
   Researched answer:
 
+Super is used within the constructor method to pass attributes from the constructor in the parent class. Without the super method the attributes would not be inheritable.
+
 
 
 2. What is a virtual DOM?
 
   Your answer:
 
+  All I really knew about the research is that DOM stands for Document Object Model.
+
   Researched answer:
 
+React creates a virtual DOM which allows for quicker manipulation then accessing the real DOM. This is used when a component is rendered and the virtual DOM checks the real DOM. This allows for only parts that require a change to render instead of the whole DOM.
+
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to display "Hello World" in the browser?
 
   Your answer:
 
+You need to have a render and a return. You must also have a react fragment enclosing code that you want rendered.
```

Comment:
This is a bit simplistic. You would also need an import, a class with a name, a render, a return and an export!